pkgs/top-level/python-packages.nix

```diff
@@ -14066,11 +14066,11 @@ in modules // {
   };
 
   mock = buildPythonPackage (rec {
-    name = "mock-1.3.0";
+    name = "mock-2.0.0";
```

Comment:
yeah okay, you don't have to keep it running if it is enormous. It's kind of what I expected with an upgrade of mock and functools.